assets/reader-activation/auth.scss

```diff
@@ -0,0 +1,212 @@
+.newspack-reader-account-link a,
+.menu-item .newspack-reader-account-link a {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	span {
+		display: block;
+		svg {
+			display: block;
+		}
+	}
+	.newspack-reader-account-link-icon {
+		height: 0;
+		margin-top: -24px;
+	}
+	.newspack-reader-account-link-label {
+		margin-left: 0.2rem;
+		@media screen and ( max-width: 959px ) {
+			border: 0;
+			clip: rect( 1px, 1px, 1px, 1px );
+			clip-path: inset( 50% );
+			height: 1px;
+			margin: -1px;
+			overflow: hidden;
+			padding: 0;
+			position: absolute !important;
+			width: 1px;
+			word-wrap: normal !important;
+		}
+	}
+}
+
+ul.newspack-reader-account-menu,
+li.menu-item .newspack-reader-account-link {
+	@media screen and ( max-width: 959px ) {
+		display: none;
+	}
+}
+.mobile-account-link {
```

Comment:
All these classes should be name-spaced to avoid collisions. We loosely use [BEM convention](https://css-tricks.com/bem-101/), here the class should be something like `.newspack-reader__mobile-account-link`. In Sass, selectors can inherit using the `&` operator, so most of the styling can be wrapped in a top-level `.newspack-reader__{}` block.

Reply:
4db09b7098bd68552ac484bc653200837335015b refactors to use BEM convention.

Reply:
There still are a few generic classnames used: `message`, `success`, `error`, `visible`. These should also be updated.

Reply:
2f624ae339eac5d14bba2b26146aaaf30f967b9a clears all non-BEM class names. It also implements some helpers and closures to make the maintenance of BEM prefixes a bit less annoying 😅